app/src/routes/index.tsx

```diff
@@ -68,7 +68,6 @@ export function getRoutes(roles: string[], t: TFunction, userRole: UserRole): Ro
           title: t('User Management'),
           key: 'user-management',
           isHomePageLink: true,
-          url: URL_USER,
```

Comment:
Note that this url is used by the navigational cards on the home dashboard. Each item on here that has a `isHomePageLink` shows up as a navigational card on the home dashboard